app/experimenter/nimbus-ui/src/lib/visualization/utils.tsx

```diff
@@ -47,25 +47,14 @@ export const getTableDisplayType = (
   return displayType;
 };
 
+// Ensures the control branch is first
 export const getSortedBranches = (analysis: AnalysisData) => {
```

Comment:
This function is likely going to be changed when we fix #6129, but I changed it here because it seems like my change from `analysis?.overall` to `analysis.overall` caused a line coverage complaint on what was previously L68. 

I don't feel we need a test for that since we should never hit what was previously on L68/what would be returned from what is now `|| {}` because of our `analysisAvailable` check, which checks for truthiness of `analysis.overall` or `analysis.weekly` and redirects the user if one of these isn't available. We could also add this utils file to the ignore list like we did the experimenter utils file, but seems like it's been okay up until this point leaving it in the coverage.